tests/unit/algo/classification/test_otx_dino_v2.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import torch
+from otx.algo.classification import DINOv2
+
+
+class TestDINOv2:
+    def setup(self) -> None:
+        self.model = DINOv2(
```

Comment:
From we observed in https://github.com/openvinotoolkit/training_extensions/actions/runs/7137703253/job/19438115260?pr=2708, we cannot allow the model download from the external URL in the **"unit test"**. Please mock `torch.hub.load()`.